src/main/java/carracing/Race.java

```diff
@@ -0,0 +1,27 @@
+package carracing;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Race {
+
+    public static void raceStart(int carCnt, int tryCnt) {
+
+        List<Car> carList = Stream.generate(Car::new)
+                .limit(carCnt)
+                .collect(Collectors.toList());
+
+        for(int i = 0; i <tryCnt; i++) {
```

Comment:
그리 중요한 부분은 아니나 IDE의 자동 정렬 기능 등을 사용하도록 습관을 갖춰주시면 좋겠네요~!
```suggestion
        for(int i = 0; i < tryCnt; i++) {
```